test/passing/extensions.ml

```diff
@@ -23,3 +23,10 @@ let invariant t =
        , xxxxxxxxxxxxxx
        , xxxxxxxxxxx
        , xxxxxxxxxxxxxxxxxxxx ) when a < b]
+
+[%%ext
+;; 11111111111111111111
+
+;; 22222222222222222222
+
+;; 33333333333333333333]
```

Comment:
@hhugo, does this fix the problematic case you ran into with #181 ?

Reply:
I believe so

Reply:
Could we special case when there is one item only?

Reply:
#195 ?